staging/all-in-one.tf

```diff
@@ -0,0 +1,65 @@
+terraform {
+  backend "s3" {
+    bucket = "tb-terraform-state"
+    key    = "terraform-staging.tfstate"
+    region = "us-east-1"
+    endpoint = "https://nyc3.digitaloceanspaces.com"
+    skip_credentials_validation = true
+    skip_get_ec2_platforms = true
+    skip_requesting_account_id = true
+    skip_metadata_api_check = true
+  }
+}
```

Comment:
has this been working? do they have a way to do state locking on DO? I kind of assume not since DO doesn't have a hosted database of any kind...

Reply:
wat

Reply:
https://www.terraform.io/docs/state/locking.html